.travis.yml

```diff
@@ -0,0 +1,12 @@
+language: node_js
```

Comment:
Testing will be done in Travis and it shows up at the top of the README if things are failing